packages/graphics/src/utils/buildRoundedRectangle.ts

```diff
@@ -71,6 +71,12 @@ function quadraticBezierCurve(
         x = getPt(xa, xb, j);
         y = getPt(ya, yb, j);
 
+        // Handle case when first curve points overlaps and earcut fails to triangulate
+        if (i === 0 && points[points.length - 2] === x && points[points.length - 1] === y)
```

Comment:
Doing that type of comparison here is unnecessary. The end-point of one curve and start point of another are not susceptible to floating-point issues because of how the rounded rect is constructed. This only occur if they are the same literal points.